### DISCUSSION TITLE
Layout XML cannot find org.imaginativeworld.whynotimagecarousel.ImageCarousel

### DISCUSSION THREAD

**Discussion by CodeMonkeyEric** [ImaginativeShohag/Why-Not-Image-Carousel#60] [Q&A] [answered]
What i did

1. Added mavenCentral() to parent gradle
2. Added `implementation 'com.google.android.material:material:1.4.0'
    implementation 'me.relex:circleindicator:2.1.6'
    implementation 'org.imaginativeworld.whynotimagecarousel:whynotimagecarousel:2.0.5'`
3. Go to layout file try to insert a view 
`<org.imaginativeworld.whynotimagecarousel.ImageCarousel
    android:id="@+id/carousel"
    android:layout_width="match_parent"
    android:layout_height="256dp" />` but cannot found.

I tried to insert view "me.relex.something", it works

Can you help?

0. My application min sdk is 21
1.  I used "androidx.appcompat:appcompat"
2. I update the styles to MaterialComponents in my styles.xml

Thank you

**Comment by ImaginativeShohag** [comment, marked as answer]
Try:
1. Build > Clean Project
2. Build > Rebuild Project
3. File > Invalidate Caches / Restart > Invalidate and Restart

It should solve you problem.

**Reply by RobiulAwalSabbib** [reply]
@ImaginativeShohag 
`Class referenced in the layout file, org.imaginativeworld.whynotimagecarousel.ImageCarousel, was not found in the project or the libraries  If a class is referenced in the manifest or in a layout file, it must also exist in the project (or in one of the libraries included by the project. This check helps uncover typos in registration names, or attempts to rename or move classes without updating the XML references properly.  Issue id: MissingClass https://developer.android.com/guide/topics/manifest/manifest-intro.html` 
i can not solve this

**Reply by RobiulAwalSabbib** [reply]
@ImaginativeShohag 
i have done 
Build > Clean Project
Build > Rebuild Project
File > Invalidate Caches / Restart > Invalidate and Restart
but still xml file show above error and not use

**Reply by RobiulAwalSabbib** [reply]
@CodeMonkeyEric 
i can not understand this
"Added mavenCentral() to parent gradle"
